ros/src/models/chrono/ros_chrono/src/steering_controller.cpp

```diff
@@ -128,8 +132,8 @@ VisualizationType tire_vis_type = VisualizationType::NONE;
 ////std::string path_file("paths/straight.txt");
 ////std::string path_file("paths/curve.txt");
 ////std::string path_file("paths/NATO_double_lane_change.txt");
-std::string path_file("paths/ISO_double_lane_change.txt");
 
+std::string path_file("paths/ISO_double_lane_change.txt");
```

Comment:
this is a steering_controller function, it should not depend on specific data. We should be providing this data to the function. Otherwise it should be called demo_steering_controller.cpp

Reply:
This line need to be deleted. The path files is loaded but not used afterward.

Reply:
Good, thanks!